### ISSUE TITLE
Split portions of MW data for easier handling

### ISSUE THREAD

**Issue by Andhrabharati** [sanskrit-lexicon/MWS#99]
I was thinking of a way to identify & handle the issues in MW99 once for all in a better manner; and it struck me that a surefire way would be to handle it **in parts**.

Hence had split the full data into identifiable portions, before starting the work on the Annexure work.

I was on some important personal work, and could not spend time in my promised work in the last week.

Now pushing these split portions of data, and guess @funderburkjim could have a look and give his comments. [I will be posting my remarks about each of these split portions sometime later.] **These are in a separate folder "extracted_section-wise_entries_Andhrabharati" inside the MWS folder.**

I will be starting the Annexure work from tomorrow, as I am now free from other things.

**Comment by gasyoun** [comment]
> I will be starting the Annexure work from tomorrow, as I am now free from other things.

We are lucky to have you around.

**Comment by Andhrabharati** [comment]
Here is the working level files, generated from the full data files above. ["unique_section-wise_entries_Andhrabharati"]

We can easily get the corrections from these unique word-lists, esp. (a) looking at the comma and dot endings (belonging to `<abbr>` and `<ls>` categories, (b) Capital letters in the beginning or middle, for the <`ls> `and `<ns>` (or `<slp1>`) type, ...

Many such easily identifiable material in lying in these files. And we can reduce the unique_words file further by looking for the English prefixes and suffixes(this is done to some extent to get the unique_words_1 file).

The other portions of the data (Metalines and the Entry words, preceding the pipe character (¦) are not given here.
which need to be corrected for `<pc>` numbers as a minimum in the meta-lines and the `<hom>` numbering in the Entry words.

This covers the full data corrections in the mw data (txt or xml) on toto.

Next possible (necessary) work is to read with the book, to identify other differences esp. for the groupings etc.

**Comment by gasyoun** [comment]
> This covers the full data corrections in the mw data (txt or xml) on toto.

My humble thanks. Let @funderburkjim come.

**Comment by Andhrabharati** [comment]
glad you liked the idea, @funderburkjim.

**Comment by Andhrabharati** [comment]
actually there should be two folders only, extracted & unique.

when I renamed the folders with Andhrabharati at the end, two more went to Github.

how two remove/delete two of these four folders?

**Comment by funderburkjim** [comment]
Just get them as you want in local installation,  then add, commit, push  (or the equivalent with
desktop github).

**Comment by funderburkjim** [comment]
P.S. -- be sure to 'pull'  my changes to your local computer before rearranging.

**Comment by Andhrabharati** [comment]
Yes; just followed your last two posts and now only the two intended folders are at Github.

**Comment by gasyoun** [comment]
@Andhrabharati how do you feel @AnnaRybakovaT can be of help here?

**Comment by Andhrabharati** [comment]
I feel she can look at the "unique words extracted" file, and filter out the probable Sanskrit or AS words from the list, and also look for bad spellings in the English words.

Two birds at a shot!

This is a major work pending in MW, as I see, as far as "pure data" is concerned.

**Comment by Andhrabharati** [comment]
If @AnnaRybakovaT is interested in this work, I can give the file location or post it directly here itself.

The file has about 50K words listed, but it is not too difficult to sift them as suggested above.

**Comment by funderburkjim** [comment]
@AnnaRybakovaT   Hi!   Nice to have you back for a while.  

Here is link to [unique words extracted file](https://github.com/sanskrit-lexicon/MWS/blob/master/mws_issue_99/unique_section-wise_entries_Andhrabharati/unique%20words%20extracted.txt).

I think some preprocessing is in order before dumping this list into Anna's lap.

For instance:
1.  remove the ending punctuation (comma, period, semicolon) -- this removes many false duplicates.
2. Preprocess with an English language dictionary -- If a word is found in dictionary, assume it is ok.  This would
    greatly reduce the number of words in the list that need to be considered.
3. Remove Arabic words

For the remaining words, we should also list the MW entry (L and headword and maybe page) 
where each word is found.   The result would be a manageable list that Anna could examine for correction.

@Andhrabharati   @gasyoun   What do you think of this plan?

**Comment by Andhrabharati** [comment]
Nice to see you off-track from the boesp-work, @funderburkjim , for @AnnaRybakovaT.

Yes, you know how to make things clear for others (I'm a different "being" altogether!).

Pl. go ahead with what you mentioned.

**Comment by gasyoun** [comment]
@funderburkjim I fully support your approach, but would add:

4. Numbers and sequences of numbers:
```
6/7
 60
 60,
 60,000
 60.
 600
 600,000
```

Otherwise all I ask is a clean list for @AnnaRybakovaT to start with. She was spening around 2 hours per day before and is ready to resume untill Greece gets some warmth again.

**Comment by Andhrabharati** [comment]
This served as a base for some cleaning up of non-English words, by Anna,

As nothing more is to be done (apparently), this issue is closable now.

**Anything (and everything) that needs to be done will be done in my revision work of MW.**

**Comment by funderburkjim** [comment]
>  Anything (and everything) ...

That sounds laudable.  Note that my concern regards your tendency to add new markup, or new uses of existing markup (such as `<lang>`). Hope you will keep this in mind if you decide to share your work with cdsl.
